src/meta-srv/src/election/postgres.rs

```diff
@@ -198,11 +272,51 @@ impl Election for PgElection {
     }
 
     async fn campaign(&self) -> Result<()> {
-        todo!()
+        let mut keep_alive_interval =
+            tokio::time::interval(Duration::from_secs(META_KEEP_ALIVE_INTERVAL_SECS));
```

Comment:
Should we use `MissedTickBehavior::Skip`?
```suggestion
            tokio::time::interval(Duration::from_secs(META_KEEP_ALIVE_INTERVAL_SECS))
                .set_missed_tick_behavior(MissedTickBehavior::Skip);
```

Reply:
etcd uses `.set_missed_tick_behavior(MissedTickBehavior::Delay);`, so I changed it to `Delay`.